pilot/pkg/model/service.go

```diff
@@ -129,6 +129,8 @@ const (
 	Passthrough
 	// DNSRoundRobinLB implies that the proxy will resolve a DNS address and forward to the resolved address
 	DNSRoundRobinLB
+	// NodeLocal implies that the proxy will only forward to node local endpoints
+	NodeLocal
```

Comment:
Using resolution is a little hacky,

Reply:
Any suggestions?

Reply:
how about add a NodeLocal bool field in ServiceAttributes

Reply:
Updated